test/linter.js

```diff
@@ -1,101 +1,101 @@
 
         function A() {
-            this.parent = 'overridden';
+            this.parent = "overridden";
         }
 
         function B() {
-            this.own = 'overridden'
+            this.own = "overridden"
         }
-        var c = { parent: 'orig', own: 'orig' };
+        var c = { parent: "orig", own: "orig" };
 
         util.inherits(B, A);
 
-        assert.deepEqual({ parent: 'orig', own: 'overridden' },
+        assert.deepEqual({ parent: "orig", own: "overridden" },
                          linter.merge(new B(), c));
     });
 });
 
-suite('preprocessScript', function () {
-    test('removes leading BOM', function () {
+suite("preprocessScript", function () {
+    test("removes leading BOM", function () {
 
-        assert.equal('var x=1;', linter.preprocessScript('var x=1;'));
+        assert.equal("var x=1;", linter.preprocessScript("var x=1;"));
 
-        assert.equal('var x=1;', linter.preprocessScript('\uFEFFvar x=1;'));
+        assert.equal("var x=1;", linter.preprocessScript("\uFEFFvar x=1;"));
 
     });
-    test('removes shebang', function () {
+    test("removes shebang", function () {
 
-        assert.equal('\nvar x=1;', linter.preprocessScript('#!/usr/bin/env node\nvar x=1;'));
+        assert.equal("\nvar x=1;", linter.preprocessScript("#!/usr/bin/env node\nvar x=1;"));
 
     });
 });
 
-suite('lint', function () {
+suite("lint", function () {
     var oldHome = process.env.HOME;
     teardown(function () {
         process.env.HOME = oldHome;
     });
 
-    test('basic lint step', function () {
+    test("basic lint step", function () {
 
-        var script = "// only a comment\n",
-            options = {edition: 'latest'},
-            JSLINT = nodelint.load(options.edition),
-            result;
+        var script = "// only a comment\n";
+        var options = {edition: "latest"};
+        var JSLINT = nodelint.load(options.edition);
+        var result;
 
         // don't let user's config interfere with our test
-        process.env.HOME = '';
+        process.env.HOME = "";
 
         result = linter.doLint(JSLINT, script, options);
 
         assert.deepEqual(result.errors, []);
         assert.ok(result.ok);
     });
 
-    test('lint finds error', function () {
+    test("lint finds error", function () {
 
-        var script = "//TODO: remove this\n",
-            options = {edition: '2013-09-22'},
-            JSLINT = nodelint.load(options.edition),
-            result;
+        var script = "//TODO: remove this\n";
+        var options = {edition: "2013-09-22"};
+        var JSLINT = nodelint.load(options.edition);
+        var result;
 
         // don't let user's config interfere with our test
-        process.env.HOME = '';
+        process.env.HOME = "";
 
         result = linter.doLint(JSLINT, script, options);
 
         assert.strictEqual(1, result.errors.length);
-        assert.strictEqual("Unexpected TODO comment.",
+        assert.strictEqual("Unexpected TODO comment.";
```

Comment:
You shouldn't change it.

Reply:
I see what you mean. I changed the semicolon back to a comma.